js/src/layouts/Project.tsx

```diff
@@ -1,43 +1,22 @@
-import React, { FC, useState } from "react";
-import { Layout, Drawer } from "antd";
+import React from "react";
+import { Layout } from "antd";
 import { useParams } from "react-router"
 
 import { ProjectSideMenu } from "../features/project/components/ProjectSideMenu"
 
-import "./Main.scss";
+import "./Project.scss";
 
-const { Sider, Content } = Layout;
+const { Content, Header } = Layout;
 
-export const Project: FC = ({ children }) => {
+export const Project: React.FC = ({ children }) => {
   const {slug} = useParams<{ slug: string }>()
 
-  const [collapsed, setCollapsed] = useState(false);
-
-  const [menuVisible, setMenuVisible] = useState(false);
-
-  const onClose = () => setMenuVisible(false);
-
   return (
-    <Layout className="app-layout">
-      <Drawer
-        className="app-mobile-drawer"
-        placement="left"
-        closable={false}
-        onClose={onClose}
-        visible={menuVisible}
-      >
-        <ProjectSideMenu slug={slug}/>
-      </Drawer>
-      <Sider
-        width={200}
-        className="app-sider"
-        collapsible
-        collapsed={collapsed}
-        onCollapse={setCollapsed}
-      >
+    <Layout className="project-layout">
+      <Header className="project-header">
         <ProjectSideMenu slug={slug}/>

```

Comment:
If it is in Header why you named it ProjectSideMenu ?

Reply:
done